src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -131,74 +89,29 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             }
         });
 
-        container.Children().Append(swapChainPanel);
-        container.Children().Append(_scrollBar);
-        Controls::Grid::SetColumn(swapChainPanel, 0);
-        Controls::Grid::SetColumn(_scrollBar, 1);
-
-        Controls::Grid root{};
-        Controls::Image bgImageLayer{};
-        root.Children().Append(bgImageLayer);
-        root.Children().Append(container);
-
-        _root = root;
-        _bgImageLayer = bgImageLayer;
-
-        _swapChainPanel = swapChainPanel;
-        this->Content(_root);
-
-        _ApplyUISettings();
-
-        // These are important:
-        // 1. When we get tapped, focus us
-        _tappedRevoker = this->Tapped(winrt::auto_revoke, [this](auto&, auto& e) {
-            Focus(FocusState::Pointer);
-            e.Handled(true);
-        });
-        // 2. Make sure we can be focused (why this isn't `Focusable` I'll never know)
-        this->IsTabStop(true);
-        // 3. Actually not sure about this one. Maybe it isn't necessary either.
-        this->AllowFocusOnInteraction(true);
-
-        // DON'T CALL _InitializeTerminal here - wait until the swap chain is loaded to do that.
-
         // Subscribe to the connection's disconnected event and call our connection closed handlers.
         _connectionStateChangedRevoker = _connection.StateChanged(winrt::auto_revoke, [this](auto&& /*s*/, auto&& /*v*/) {
             _ConnectionStateChangedHandlers(*this, nullptr);
         });
 
-        _root.AllowDrop(true);
-        _root.Drop({ get_weak(), &TermControl::_DragDropHandler });
-        _root.DragOver({ get_weak(), &TermControl::_DragOverHandler });
+        _ApplyUISettings();
     }
 
     // Method Description:
-    // - Create the SearchBoxControl object, and attach it
-    //   to the Terminal Control root
-    // Arguments:
-    // - <none>
-    // Return Value:
-    // - <none>
+    // - Loads the search box from the xaml UI and focuses it.
     void TermControl::CreateSearchBoxControl()
     {
-        if (!_searchBox)
+        // Lazy load the search box control.
+        if (auto loadedSearchBox{ FindName(L"SearchBox") })

```

Comment:
Just to be sure, this will return the search box for _this_ TermControl right? If we have multiple panes with their own search boxes, there won't be any ambiguity?

Reply:
correct. FindName is a member on `this`.